io.dbeaver.DBeaverCommunity.appdata.xml

```diff
@@ -31,6 +31,7 @@
         </screenshot>
     </screenshots>
     <releases>
+        <release version="21.0.1" date="2021-03-21"/>
```

Comment:
This duplicate is causing the build to fail

Reply:
Removing the previous `21.0.1` version should solve the problem. I think it's important to let this pull request pass, because next versions then the repository is completely compatible with the external PRs

Reply:
Let's try it :)